blockbase/block-template-parts/header-arbutus.html

```diff
@@ -0,0 +1,27 @@
+<!-- wp:group {"className":"gapless-group","layout":{"inherit":"true"}} -->
```

Comment:
I think we can achieve the differences in this file using theme.json

Reply:
In general we want a gap between items, we only want SOME groups to be "gapless".  I'm not sure how that could be achieved in theme.json.  

Alternately we could just leave the gap and adjust the size of the spacer to be it's current size minus the gap size.  The only problem with that was that it looked a little strange as an adjusting mechanism with the gap there, and also there are a couple of themes where the "header gap" is smaller than the "block gap" and the design target couldn't be achieved.  If it's OK making those gaps slightly larger than they are now then we could do away with the concept entirely.